<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerOperationWidget</class>
 <widget class="QWidget" name="TimerOperationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change item</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Select Timer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioTimer1">
        <property name="text">
         <string>Timer 1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">groupTimer_arg5</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioTimer2">
        <property name="text">
         <string>Timer 2</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">groupTimer_arg5</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Operation</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioOpTime">
        <property name="text">
         <string>Set Time</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">groupOp_arg0</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioOpStart">
        <property name="text">
         <string>Start Timer</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">groupOp_arg0</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioOpEnd">
        <property name="text">
         <string>Stop Timer</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">groupOp_arg0</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOperand">
     <property name="title">
      <string>Operand</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="TimerOperandWidget" name="operand_arg1" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkOptionShowTimer_arg3">
        <property name="text">
         <string>Show Timer on Screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkOptionBattleTimer_arg4">
        <property name="text">
         <string>Continue Timer in Battle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>191</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimerOperandWidget</class>
   <extends>QWidget</extends>
   <header>ui/common/operand_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>helpRequested()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="groupTimer_arg5"/>
  <buttongroup name="groupOp_arg0"/>
 </buttongroups>
</ui>
